react/components/ModalContent.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import { useCssHandles } from 'vtex.css-handles'
+
+interface Props {
+  className?: string
+}
+
+const CSS_HANDLES = [
+  'contentContainer'
+]
+
+
+const ModalContent: React.FC<Props> = props => {
+  const { className = '', children } = props
```

Comment:
Is it necessary to receive `className` as a regular prop?

Reply:
I think it is because I'm generating the paddings tachyons classes in the `Modal` component, so I prefer creating the `className` prop instead of creating something like `paddingClasses` prop